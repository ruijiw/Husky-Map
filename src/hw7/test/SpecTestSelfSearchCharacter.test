# Find a path using the LOLDictionary dataSet.
# This test uses the parser, the MarvelPaths2's setUpGraph
# and Dijkstra algorithm and applies to a targeted dataSet.
# This test is for the user asks for the character to itself case.

LoadGraph dictionary LOLDictionary.tsv

FindPath dictionary TEEMO TEEMO

FindPath dictionary JINX JINX