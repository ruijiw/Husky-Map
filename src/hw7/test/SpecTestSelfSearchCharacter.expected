# Find a path using the LOLDictionary dataSet.
# This test uses the parser, the MarvelPaths2's setUpGraph
# and Dijkstra algorithm and applies to a targeted dataSet.
# This test is for the user asks for the character to itself case.

loaded graph dictionary

path from TEEMO to TEEMO:
total cost: 0.000

path from JINX to JINX:
total cost: 0.000